docs/usage/self-hosted-configuration.md

```diff
@@ -956,6 +956,10 @@ For example: `:warning:` will be replaced with `⚠️`.
 Some cloud providers offer services to receive metadata about the current instance, for example [AWS Instance metadata](https://docs.aws.amazon.com/AWSEC2/latest/WindowsGuide/ec2-instance-metadata.html) or [GCP VM metadata](https://cloud.google.com/compute/docs/metadata/overview).
 You can control if Renovate should try to access these services with the `useCloudMetadataServices` config option.
 
+## useSqliteCache
+
+For improved performance, set this to `true` and Renovate will use a SQLite database for its package cache instead of the local file system.
```

Comment:
we should mention that this is mutual exclusive to `redisUrl`. redis would be preferred over sqlite if both are configured.